edit-post/components/header/index.js

```diff
@@ -78,18 +64,16 @@ function Header( {
 	);
 }
 
-export default connect(
-	( state ) => ( {
-		isGeneralSidebarEditorOpen: getOpenedGeneralSidebar( state ) === 'editor',
-		isPublishSidebarOpen: isPublishSidebarOpened( state ),
-		hasActiveMetaboxes: hasMetaBoxes( state ),
-		isSaving: isSavingMetaBoxes( state ),
-	} ),
-	{
-		onOpenGeneralSidebar: () => openGeneralSidebar( 'editor' ),
-		onCloseGeneralSidebar: closeGeneralSidebar,
-		onTogglePublishSidebar: togglePublishSidebar,
-	},
-	undefined,
-	{ storeKey: 'edit-post' }
+export default compose(
+	withSelect( ( select ) => ( {
+		isEditorSidebarOpened: select( 'core/edit-post' ).isEditorSidebarOpened(),
```

Comment:
Not directly related to this pr, and even if we decide to some change it should not be in this PR. We are always repeating select( 'core/edit-post' ). This  makes me wonder if we should not do something like: const editPostSelect = select( 'core/edit-post' ). Or add an argument to withSelect like withSelect( 'core/edit-post', ( select )...

Reply:
Actually, it has been discussed a few times in the past. I will find some link and share on Monday :)

Reply:
@aduth I want to refactor all connect occurrences next week to see if it has any impact on the edit-post module’s bundle size. I’ll follow your advice despite the fact that I personally like this short notation 😃